### DISCUSSION TITLE
Gestures don't work if another touch is moving?

### DISCUSSION THREAD

**Discussion by magikarp28** [pmndrs/use-gesture#421] [Q&A] [answered]
Hello, I've been using this @use-gesture for some of my touchscreen projects and have noticed gestures sometimes won't initiate.
I've finally figured out when this happens.
If I move my finger around and then try to initiate any kind of gesture it does not work. Interestingly the gestures don't stop working if they are initiated before the touch events. Also the problem appears only when touches are moving, not if it is just a touchdown.
I can replicate this problem with anything using the library, for example in this official example https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/card-zoom

Why is this happening and could it be fixed?

**Comment by dbismut** [comment]
I'm not sure I understand properly. Would there be a way to capture this on video?

What I can tell you though is that you need a `touchstart` or `pointerdown` event on the target element for the pinch to start. If you start the touch outside the element for which you bind the handlers, the gesture won't start. Do you think this could be related?

**Reply by magikarp28** [reply]
Yes its kind of hard to explain so I should have just made a video from the start.
Take a look, hopefully it makes it clear. It seems like really odd Behaviour.

https://youtu.be/OQ2dkxVDN3E

Its just the unchanged example I linked.

**Reply by magikarp28** [reply]
Could I get some feedback on this?
My testing would suggest that the issue is that usegesture uses pointers instead of touch events however since `pointer: { touch: true } `does not resolve the issue maybe I'm wrong.

**Reply by dbismut** [reply]
Sorry. Are you able to reproduce this on a regular phone?

**Reply by magikarp28** [reply]
The issue is present on the phone however `pointer: { touch: true }` fixes it.
It seems quite odd that it works for a phone but not a windows browser (my target device). 
Note that this issue does not exist in other libs that use swiping gestures like a carousel so it does not seem to be a browser bug.

**Reply by dbismut** [reply]
What hardware are you using? And what is the phone?

**Reply by magikarp28** [reply]
I made a mistake for not testing on firefox. It actually works on firefox but not on anything chromium based.

I don't think it has anything to do with harware but I'm using an iiyama prolite TF6538UHSC touch monitor and I've tested with a few different intel nuc devices, for example NUC8i5BEK.

**Reply by dbismut** [reply]
What about the phone? In any case it will be hard for me to debug this without accessing the hardware, but we can try a debug session together on Discord if you'd like.

**Reply by magikarp28** [reply]
The phone is xiaomi mi 9T running on MIUI 12.1.4 and android 11 but it works as expected on the phone.

I would be open to trying to solve this over discord but I'm not sure how much of a help I could be. You don't have access to a windows machine with a touchscreen?
How can I contact you?

**Reply by dbismut** [reply]
No I don't unfortunately. You can reach me on Discord, the link is on this repo homepage. 

On my iPhone the sandbox works fine, even without `pointer.touch`.

**Comment by dbismut** [comment, marked as answer]
10.2.5 is out and fixes it!
